aten/src/ATen/native/cpu/BinaryOpsKernel.cpp

```diff
@@ -965,7 +965,7 @@ void mse_kernel(TensorIteratorBase& iter) {
         "This may be slower than using float or double-type tensors.");
   }
 
-  AT_DISPATCH_FLOATING_TYPES_AND_HALF(iter.dtype(), "mse_cpu", [&]() {
+  AT_DISPATCH_FLOATING_TYPES_AND2(kHalf, kBFloat16, iter.dtype(), "mse_cpu", [&]() {
```

Comment:
Do we have UT to cover the BF16 path?

Reply:
Good point, let's wait for CI signal, if it'll green(which it should not), I'll modify op_db to test on bf16

Reply:
And there are, but for some reason they are not running on this PR, but on the followup, see https://github.com/pytorch/pytorch/actions/runs/11715233003/job/32631645140